minijail/rust/minijail-sys/libminijail.rs

```diff
@@ -387,3 +387,15 @@ extern "C" {
 extern "C" {
     pub fn minijail_log_to_fd(fd: ::std::os::raw::c_int, min_priority: ::std::os::raw::c_int);
 }
+extern "C" {
```

Comment:
We should get `bindgen` to work in our minijail "fork". Hm. Out of scope for this PR but the correct way. @pclesr can you create a ticket for that: `Use bindgen for libminijail.rs generation`. Maybe there's already a suitable bindgen call in the minijail crate.

Reply:
Will do